src/sentry/workflow_engine/models/workflow.py

```diff
@@ -38,8 +38,20 @@ class Workflow(DefaultFieldsModel, OwnerModel, JSONConfigBase):
 
     @property
     def config_schema(self) -> dict[str, Any]:
-        # TODO: fill in
-        return {}
+        return {
+            "$schema": "https://json-schema.org/draft/2020-12/schema",
+            "$id": "https://example.com/product.schema.json",
+            "title": "Workflow Schema",
+            "type": "object",
+            "properties": {
+                "frequency": {
```

Comment:
wondering if `environment_id` should also be moved here if it's optional, but the FK aspect of having it as a field is nice